db/migrate/20220124095341_drop_connect_request_tables.rb

```diff
@@ -0,0 +1,6 @@
+class DropConnectRequestTables < ActiveRecord::Migration[6.1]
+  def change
+    drop_table :connect_requests
+    drop_table :connect_slots
+  end
```

Comment:
Use `up` and `down` instead of `change`. `raise ActiveRecord::IrreversibleMigration` in the `down` method.

Reply:
Done